src/freenas/etc/ix.rc.d/ix-syslogd

```diff
@@ -246,27 +246,25 @@ ix_syslogd_start()
 	if [ -d "${mp}/log" ]
 	then
 
-		#
-		#	Pick up any new directories and sync them
-		#
 		if [ ! -L /var/log ]; then
+
+			#
+			#	Pick up any new directories and sync them
+			#
 			for dir in $(find /var/log/ -type d)
 			do
 				local dst="${mp}/log/${dir#/var/log/}"
 				if [ ! -d "${dst}" ]
 				then
-					/usr/local/bin/rsync -avz ${dir}/* ${dst}/ >/dev/null 2>&1
+					/usr/local/bin/rsync -avz ${dir} ${dst}/ >/dev/null 2>&1
```

Comment:
Are you sure removing /* is the right thing here? In theory it would copy the directory and not its contents

Reply:
So no, for rsync it copies the contents recursively including files and directories. We should be good here. I am building an iso however and will test to be sure that nothing is wrong in our case

Reply:
Yes it does, but in this case i think it will also copy the top level directory, not only its contents.

Reply:
Which is desired ;) i.e /var/log/netdata to --> dataset/log/

Reply:
No

What you're doing is:

/var/log/netdata to --> /var/db/system/log/netdata/